### PR TITLE
Jw requests button bug

### PR DESCRIPTION
**Pull request by jwiel86**

### CODE REVIEW

File: app/helpers/dashboard/protocols_helper.rb [hunk 1]
```diff
@@ -47,7 +47,15 @@ def pis_display(protocol)
     protocol.principal_investigators.map(&:full_name).join ", "
   end
 
-  def archived_button_display(protocol)
-    content_tag( :button, (protocol.archived ? t(:dashboard)[:protocols][:table][:unarchive] : t(:dashboard)[:protocols][:table][:archive])+" #{protocol.type.capitalize}", type: 'button', class: 'protocol-archive-button btn btn-warning btn-sm' )
+  def display_requests_button(protocol, admin_protocols, current_user)
+    if !protocol.sub_service_requests.empty? && (!protocol.project_roles.find_by(identity: current_user).nil? || admin_protocols.try(:include?, protocol.id))
```
Comment (wtholt) [inline app/helpers/dashboard/protocols_helper.rb]: What about 

     if protocol.sub_service_requests.any?

File: app/helpers/dashboard/protocols_helper.rb [hunk 2]
```diff
@@ -47,7 +47,15 @@ def pis_display(protocol)
...
+      content_tag( :button, t(:dashboard)[:protocols][:table][:requests], type: 'button', class: 'requests_display_link btn btn-default btn-sm' )
```
Comment (wtholt) [inline app/helpers/dashboard/protocols_helper.rb]: This is a little hard to follow

    (!protocol.project_roles.find_by(identity: current_user).nil?

Comment (kyle-glick) [inline app/helpers/dashboard/protocols_helper.rb]: find_by returns nil if nothing is found, which is why this was done. Recommend protocol.project_roles.where(identity: current_user).any?
